mock-omaha-server/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2024 The Fuchsia Authors
+#
+# Licensed under a BSD-style license <LICENSE-BSD>, Apache License, Version 2.0
+# <LICENSE-APACHE or https://www.apache.org/licenses/LICENSE-2.0>, or the MIT
+# license <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+[package]
+name = "mock-omaha-server"
+version = "0.1.0"
+edition = "2021"
+description = "Mock implementation of the server end of the Omaha Protocol"
+license = "BSD-2-Clause OR Apache-2.0 OR MIT"
+repository = "https://github.com/google/omaha-client"
+readme = "README.md"
+
```

Comment:
How about adding this to prevent publishing this for now?

```
publish = false
```

Reply:
Applied in upcoming pull request.